index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+  <title>First mobile project</title>
+</head>
+<body>
+  <header>
+    <div class="toolbar">
+      <a class="my-logo" href="index.html">My Logo</a>
+      <nav>
+        <a class="bars" href="index.html"><img src="img/Union.png" alt="bars"/></a>
+      </nav>   
+    </div>
+  </header>
+  <main>
+    <section class="main-section">
+      <div class="section-content">
+        <h1>Hey There.<br>I’m Ricardo.</h1>
+        <p class="developer">I'm a Software Developer</p>
+        <p class="developer-info">I can help you build a product, feature<br>or website Look through some of my<br>work and experience! If you like what you<br>see and have a project you need coded,<br>don’t hesitate to contact me.
+        <nav>
+          <ul class="social">
+            <li><a href="https://github.com/"><img src="img/Gith.png" alt="Github"/></a></li>
+            <li><a href="https://www.linkedin.com/home/?originalSubdomain=cl"><img src="img/Lnk.png" alt="Linkedin"/></a></li>
+            <li><a href="https://www.instagram.com/"><img src="img/Hand.png"></a></li>
+            <li><a href="https://twitter.com/?lang=es"><img src="img/Tw.png" alt="Twiter"></a></li>
+            <li><a href="https://es-la.facebook.com/"><img src="img/M.png"></a></li>
+          </ul>
+        </nav>
```

Comment:
[OPTIONAL]

- [ ] I would kindly suggest you remove the social media list icon from being wrapped inside of the `nav` tag because they are not nav-links, It would be better if you replace it with a `div` tag

Reply:
Thanks for your suggestions. Have a nice day..